src/module.indexer/model/active.price.ts

```diff
@@ -0,0 +1,86 @@
+import { Injectable } from '@nestjs/common'
+import { Indexer, RawBlock } from '@src/module.indexer/model/_abstract'
+import { OraclePriceActive, OraclePriceActiveMapper } from '@src/module.model/oracle.price.active'
+import { OraclePriceAggregated, OraclePriceAggregatedMapper } from '@src/module.model/oracle.price.aggregated'
+import { PriceTickerMapper } from '@src/module.model/price.ticker'
+import { HexEncoder } from '@src/module.model/_hex.encoder'
+import { PriceTicker } from '@whale-api-client/api/prices'
+import BigNumber from 'bignumber.js'
+
+const DEVIATION_THRESHOLD = 0.3
+const BLOCK_INTERVAL = 120
+const MINIMUM_LIVE_ORACLES = 2
+
+@Injectable()
+export class ActivePriceIndexer extends Indexer {
+  constructor (
+    private readonly aggregatedMapper: OraclePriceAggregatedMapper,
+    private readonly activePriceMapper: OraclePriceActiveMapper,
+    private readonly priceTickerMapper: PriceTickerMapper
```

Comment:
is that good to have network differentiate the `BLOCK_INTERVAL` ??  
120 mainnet | 6 regtest
:thinking: 
```suggestion
    private readonly priceTickerMapper: PriceTickerMapper
    @Inject('NETWORK') protected readonly network: NetworkName
```

Reply:
any other suggestions?

Reply:
Oh right I got you, yes will integrate this